src/server/pfs/cmds/cmds.go

```diff
@@ -357,22 +357,30 @@ Files can be read from finished commits with get-file.`,
 		}),
 	}
 
-	var mountPoint string
 	mount := &cobra.Command{
-		Use:   "mount [repo/commit:alias...]",
+		Use:   "mount path/to/mount/point",
 		Short: "Mount pfs locally.",
 		Long:  "Mount pfs locally.",
-		Run: pkgcobra.Run(func(args []string) error {
+		Run: func(c *cobra.Command, args []string) {
 			//lion.SetLevel(lion.LevelDebug)
 			apiClient, err := getAPIClient(address)
 			if err != nil {
-				return err
+				fmt.Fprintf(os.Stderr, "%v\n", err)
+				os.Exit(1)
 			}
 			mounter := fuse.NewMounter(address, apiClient)
-			return mounter.Mount(mountPoint, shard(), parseCommitMounts(args), nil)
-		}),
+			if len(args) != 1 {
+				c.Usage()
+				os.Exit(1)
+			}
```

Comment:
Ah, I missed that helper. I thought I had to call c.Usage() manually, and so ripped out the other one. I'll use this instead.